### PR TITLE
FIX: properly check key values for shortcuts

### PR DESCRIPTION
**Pull request by janzenisaac**
- Update angle bracket components to use vanilla attributes where possible
- Update how we check for key value after changing to `on "change"` from `onChange`

### CODE REVIEW

File: assets/javascripts/discourse-assign/connectors/groups-interaction-custom-options/assignable-interaction-fields.hbs [hunk 1]
```diff
@@ -11,7 +11,7 @@
     @valueProperty="value"
     @value={{assignableLevel}}
     @content={{assignableLevelOptions}}
-    @class="groups-form-assignable-level"
+    class="groups-form-assignable-level"
```
Comment (CvX) [inline assets/javascripts/discourse-assign/connectors/groups-interaction-custom-options/assignable-interaction-fields.hbs:14]: I don't this works. d-button has explicitly added `...attributes` to its element, but select-kit's combo box doesn't have that

Comment (davidtaylorhq) [reply assets/javascripts/discourse-assign/connectors/groups-interaction-custom-options/assignable-interaction-fields.hbs]: For classic components, attributes are added to the wrapper element (assuming `tagName` is not overridden to `""`)

File: assets/javascripts/discourse/templates/modal/assign-user.hbs [hunk 2]
```diff
@@ -60,16 +60,16 @@
 </DModalBody>
 
 <div class="modal-footer">
-  {{d-button
-    label=(if
+  <DButton
+    @label={{(if
```
Comment (jjaffeux) [inline assets/javascripts/discourse/templates/modal/assign-user.hbs:64]: Shouldnt this just be `{{if ...`

File: assets/javascripts/discourse/templates/modal/assign-user.hbs [hunk 3]
```diff
@@ -60,16 +60,16 @@
...
       model.reassign
       "discourse_assign.reassign.title"
       "discourse_assign.assign_modal.assign"
-    )
-    icon=inviteIcon
-    action=(action "assign")
+    )}}
+    @icon={{inviteIcon}}
+    @action={{(action "assign")}}
```
Comment (jjaffeux) [inline assets/javascripts/discourse/templates/modal/assign-user.hbs:70]: Same question about `{{}}` and also is it possible to do `this.assign` here instead or it doesnt work in this case?

File: assets/javascripts/discourse/templates/components/remind-assigns-frequency.hbs [hunk 4]
```diff
@@ -2,7 +2,7 @@
   <div class="controls controls-dropdown">
     <label>{{i18n "discourse_assign.reminders_frequency.description"}}</label>
     <ComboBox
-      @id="remind-assigns-frequency"
+      id="remind-assigns-frequency"
```
Comment (CvX) [inline assets/javascripts/discourse/templates/components/remind-assigns-frequency.hbs:5]: the lack of `...attributes` applies to all attrs (yeah, select-kit is not glimmer ready 😉)

File: assets/javascripts/discourse-assign/controllers/assign-user.js [hunk 5]
```diff
@@ -72,8 +72,8 @@ export default Controller.extend(ModalFunctionality, {
   },
 
   @action
-  handleTextAreaKeydown(value, event) {
-    if ((event.ctrlKey || event.metaKey) && event.key === "Enter") {
+  handleTextAreaKeydown(value) {
```
Comment (jjaffeux) [inline assets/javascripts/discourse-assign/controllers/assign-user.js:75]: So value is actually an event here right?

File: assets/javascripts/discourse-assign/controllers/assign-user.js [hunk 6]
```diff
@@ -72,8 +72,8 @@ export default Controller.extend(ModalFunctionality, {
...
+    if ((value.ctrlKey || value.metaKey) && value.key === "Enter") {
```
[the remark this answers is not in the document]
Comment (jjaffeux) [reply assets/javascripts/discourse-assign/controllers/assign-user.js:76]: just made the same comment 😅

**Review by CvX — APPROVED** [review]

**Pull request merged**